recipes/orange-canvas-core/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "orange-canvas-core" %}
+{% set version = "0.1.4" %}
+{% set sha256 = "63dc146742a02b8e182ed93520bddca930d8323e8361fde06bd3402601d5b17d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install -vv --no-deps . "
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - pip    >=18.0
+    - setuptools
+    - packaging
+    - anyqt
+    - pyqt
+    - docutils
+    - commonmark
+    - requests
+    - cachecontrol
+    # via cachecontrol[filecache]
+    - lockfile >=0.9
+
+test:
+  imports:
+    - orangecanvas.main
+    - orangecanvas.application.canvasmain
+
+about:
+  home: https://github.com/biolab/orange-canvas-core
+  license: GPL-3.0
+  license_family: GPL
+  license_file: LICENSE.txt
+  summary: 'Core component of Orange Canvas'
+
+  description: |
+    Orange Canvas Core is is a framework for building graphical user
+    interfaces for editing workflows. It is a component used to build
+    the Orange Canvas (http://orange.biolab.si) data-mining application
+    (for which it was developed in the first place).
+
+  doc_url: https://orange-canvas-core.readthedocs.io/
+  dev_url: https://github.com/biolab/orange-canvas-core
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
+    # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
+    - ales-erjavec
+    - lanzagar
+    - markotoplak
+    - primozgodec
```

Comment:
@lanzagar @markotoplak @primozgodec
Could you please confirm that you want to be a maintainer of this recipe as well?

Reply:
I am confirming at I am OK to be a maintainer

Reply:
I am confirming that I would like to be a maintainer.

Reply:
I also confirm I would like to be a maintainer.